docs/4.1/ssh_gsuite.md

```diff
@@ -33,6 +47,31 @@ like:
 
 ![Copy Client Secret](img/gsuite/gsuite-5-copy-client-id.png)
 
+## Create a Service Account
+
+![Create OAuth Creds](img/gsuite/gsuite-5a-service-account.png)
+Leave Service account users roles, and admin roles as blank.
+![Create OAuth Creds](img/gsuite/gsuite-5b-service-account.png)
+Leave Service account permissions as blank.
+![Create OAuth Creds](img/gsuite/gsuite-5c-service-account.png)
+### Enable Account Delegation:
+![Create OAuth Creds](img/gsuite/gsuite-5d-service-account-delegation.png)
+![Create OAuth Creds](img/gsuite/gsuite-5e-enable-delegation.png)
+### Download Service Account JSON
+![Create OAuth Creds](img/gsuite/gsuite-5f-download-json.png)
+
+This JSON file will need to be uploaded to the Authenication server, and will be later referenced by 
```

Comment:
Maybe add a note about it needing to be present on all auth servers in an HA config?